docs/app/Examples/modules/Dropdown/Types/AllowAdditions.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import { Dropdown } from 'stardust'
+
+export default class AllowAdditionsExample extends Component {
+  componentWillMount() {
+    this.setState({ currentValue: undefined })
+  }
+
+  handleChange = (e, value) => this.setState({ currentValue: value })
+
+  render() {
+    const { currentValue } = this.state
+
+    return (
+      <div>
+        <Dropdown
+          options={[
+            { text: 'English', value: 'English' },
+            { text: 'French', value: 'French' },
+            { text: 'Spanish', value: 'Spanish' },
+            { text: 'German', value: 'German' },
+            { text: 'Chinese', value: 'Chinese' },
+          ]}
```

Comment:
Let's pull these options into a `const` above the example.  This way we can reuse them.  Also, they'll be created once instead of every render.

Then, let's add a single and multiple dropdown that use the same options and both allow additions.  Similar to http://semantic-ui.com/modules/dropdown.html#tagging-and-user-additions.

This raised the first bug, multiple dropdowns do not allow adding multiple additions.